README.md

```diff
@@ -1,45 +1,94 @@
+
+If this excites you and you are willing to learn, you are a great fit for the
+team.  When determinng what you want to work on, the first choice is:
+- Do I want to program?
+- Do I want to design circuit boards?
+- Do I want to do CAD?
+
+After you decide that you can take a look through the [repositories](https://github.com/queens-satellite-team)
+and associated issues ([here](https://github.com/queens-satellite-team/moc-sat/issues) are the MocSat issues).
+Once you have found a project or issue that interests you:
+  - If you know how to fix the issue, do so and make a pull request to be
+    reviewed by the team lead.
+  - Post in the [new members](https://github.com/orgs/queens-satellite-team/teams/new-members)
+    team discussion with what your technical interest is (code, PCB design, CAD)
+    and sub-system. The team lead will then be able to give you more information.
+
+### The Competition
+In order to compete in the CSDC we are building a [3 Unit Cube-Satellite](https://www.asc-csa.gc.ca/eng/satellites/cubesat/what-is-a-cubesat.asp).
+We were started in 2016 and have competed once in the CSDC 2016-2018 competition
+season. The CSDC is a two-year competition, so we would have competed again in
+the 2020 but the competition was postponed due to Covid-19, likely until 2021 or
+2022.
+
+The competition is held at the Canadian Space Agency. The satellite undergoes
+vibration testing to assess flight readiness. It is then inspected by the judges
+and team members are asked to demonstrate the functionality of the required
+subsystems. The satellite that is deemed to be the most flight-ready is awarded
+1st place. Though it is the satellite that is being evaluated, the team plays an
+important role in the decision. The judges understand that the resources available
+to each team (money and faculty support) are not evenly distributed and that
+some teams have to make due with less. For example, a team who could not afford
+to purchase flight ready solar panels is not penalized against a team who could.
+They are interested in the effectiviness of the team and the ability to
+demonstrate exceptional engineering ability.
+
+The CSDC only ranks the top three teams and we didn't place in the top three
+in our 2016-2018. Anecdotally, out of about 12 teams we were in the middle of the
+pack. The team at the time was much stronger in mechanical engineering than
+electrical or computer, so we had a good structure but some of our electrical
+sub systems were not working. It is not uncommon for teams to bring unfinished
+or unintegrated satellites to the competition. In fact, having a fully integrated
+satellite will likely place you in or near the top three.
+
+### Roles on the Team
+The satellite is composed of sub-systems. The sub-systems are:
+  - Attitude Determination and Control System (ADCS)
+  - Communications System (Comms)
+  - Electrical Power System (EPS)
+  - On-Board Computer (OBC)
```

Comment:
Don't we technically have a mech team now?  I am not sure if that is worth adding.